packages/lib-user/src/components/UserHome/components/RecentSubjects/RecentSubjectsContainer.js

```diff
@@ -36,16 +35,31 @@ function RecentSubjectsContainer({ authUser }) {
 
   const recentProjectIds = [...new Set(recents?.map(recent => recent.links?.project))]
 
-  const { data: projects, isLoading: projectsLoading, error: projectsError } = usePanoptesProjects(recentProjectIds)
+  const {
+    data: projects,
+    isLoading: projectsLoading,
+    error: projectsError
+  } = usePanoptesProjects({
+    cards: true,
+    id: recentProjectIds?.join(',')
+  })
 
   // Attach project slug to each recent
   let recentsWithProjectSlugs
   if (projects?.length) {
-    recentsWithProjectSlugs = recents.map(recent => {
-      const { slug } = projects.find(project => recent.links.project === project.id)
```

Comment:
@shaunanoordin - this was the cause of the page crash. The recents included a project that was not included in the `usePanoptesProjects` projects, and when this line attempted to get the slug from nothing it caused the crash.